daq/host.py

```diff
@@ -103,6 +103,7 @@ def __init__(self, runner, device, config):
         self.device = device
         self.target_mac = device.mac
         self.target_port = device.port.port_no
+        self.nonof_dev_port = str(device.nonof_port_num) if device.nonof_port_num else None
```

Comment:
this logic seems wonky, and I'm guessing you're trying to take a shortcut with the definition of self.target_port.  Under what conditions is the meaning of "self.target_port" not the useful value here?  Seems like the concept of a target_port should apply to all switch cases (they all have the same sequential alignment of ports).

You mentioned that just setting self.target_port to something causes problems... but I would think of this the other way around and figure out what trigger to use for those cases rather than changing the foundational meaning of self.target_port, which is ultimately very confusing.

Reply:
target_port needs to be an integer, so assigning  nonof_dev_port which is a string causes an error - 
```Feb 24 12:46:47 ipaddr.ipaddr01 ERROR   Exception for 38d135021d24: '2' has type str, but expected one of: int, long ```

Raised by line `return usi.SwitchInfo(**params)` (a protobuf) in the `_buid_switch_info` function in `host.py`

Reply:
Ah. Forgot to update this in the repo. Reworked the file. Obsolete now.